Core/NoOpRunner.Core/GameWindow.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace NoOpRunner.Core
 {
-    public class GameWindow
+    public class GameWindow//GameMap ??? 
```

Comment:
Yes, probably. Again, maybe think of where obstacles/rockets etc will belong and name accordingly

Reply:
Yes